tldw_Server_API/app/api/v1/schemas/flashcards.py

```
@@ -109,6 +121,64 @@ def _populate_scheduler_settings(cls, data: Any) -> Any:
+    name: str = Field(..., min_length=1)
+    model_type: FlashcardTemplateModelType = "basic"
+    front_template: str = Field(..., min_length=1)
+    back_template: Optional[str] = None
+    notes_template: Optional[str] = None
+    extra_template: Optional[str] = None
+    placeholder_definitions: list[FlashcardTemplatePlaceholderDefinition] = Field(default_factory=list)
+
+    @model_validator(mode="after")
+    def _validate_scaffold_requirements(self) -> "FlashcardTemplateCreate":
+        if self.model_type in ("basic", "basic_reverse") and not str(self.back_template or "").strip():
+            raise ValueError("back_template is required for basic and basic_reverse templates")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)